addons/binding/org.openhab.binding.dscalarm/ESH-INF/thing/keypad.xml

```diff
@@ -51,6 +51,14 @@
 				<label>Keypad AC LED</label>
 				<description>Keypad AC LED (0=Off, 1=On, 2=Flashing)</description>
 			</channel>
+            <channel id="keypad_lcd_update" typeId="message">
```

Comment:
hm, with auto-formatter, there should be tabs instead of spaces here (as in the other lines that you didn't touch